README.md

```diff
@@ -1,72 +1,25 @@
-# A Regularized Conditional GAN for Posterior Sampling in Inverse Problems [[arXiv]](https://arxiv.org/abs/2210.13389)
+# Generative modelling for mass-mapping with fast uncertainty quantification [[arXiv]](https://arxiv.org/abs/2410.24197)
 ## Setup
 See ```docs/setup.md``` for basic environment setup instructions.
 
-## Reproducing our Results
-### MRI
-See ```docs/mri.md``` for instructions on how to setup and reproduce our MRI results.
-
-## Extending the Code
-See ```docs/new_applications.md``` for basic instructions on how to extend the code to your application.
+## Reproducing the our Results
+### 
+See ```docs/mass_mapping.md``` for instructions on how to setup and reproduce our COSMOS results.
 
```

Comment:
Could you coordinate with Matthijs so that the radio application of the GAN is also updated on this main branch. It'd be good to have everything to reproduce the results of both papers in a single repo.

Reply:
Yes, will have to wait for Matthijs to pick this up I think, once he's got some time